src/main/java/org/broadinstitute/consent/http/models/Dataset.java

```diff
@@ -471,4 +477,37 @@ public User getCreateUser() {
   public void setCreateUser(User createUser) {
     this.createUser = createUser;
   }
+
+  public List<String> getDataCustodianEmails() {
+    if (!(this.getProperties() == null)) {
+      Gson gson = GsonUtil.getInstance();
+      Type listOfStringsType = new TypeToken<List<String>>() {}.getType();
+      return this.getProperties()
+          .stream()
+          .filter(p -> (Objects.nonNull(p.getSchemaProperty()) &&
+              p.getSchemaProperty().equalsIgnoreCase("dataCustodianEmail")))
+          .filter(p -> p.getPropertyType().equals(PropertyType.Json))
+          .map(p -> {
+            List<String> stringList = gson.fromJson(p.getPropertyValueAsString(), listOfStringsType);
+            return stringList;
+          })
+          .flatMap(List::stream)
+          .distinct()
+          .toList();
+    }
+    return List.of();
+  }
+
+  public List<String> getDataDepositorEmails() {
```

Comment:
Technically, this is getting the string value of the `Data Depositor` field, which could be an email, or not. I suggest renaming it so it isn't confusing to the callers. Related to this, I think the email service handles the case where the string is NOT a valid email, but maybe an extra check in `VoteService` would prevent sending invalid data in the first place.